### DISCUSSION TITLE
feature: class spawn points with instances maps

### DISCUSSION THREAD

**Discussion by NikoCummings** [AscensionGameDev/Intersect-Engine#2046] [Ideas]
### Description

The ability to set a class spawn points to an instanced map

### Intended Use-Case

Alot of games are doing story book intro cut scenes or having in game cut scenes where NPCs are doing move routes with dialogue etc. it would be nice to be able to set a classes spawn point to an instanced map where it is just the player and the cut scene. I would imagine this could cut down some lag too, especially on a games launch where many players are signing up and logging in at once.

### Duplicate Check

- [X] This feature request is not a duplicate to the best of my knowledge.

**Comment by da-rulez** [comment]
I think this would create _more_ lag if you make an instance for every new player on game launch. 

That said, can't you immediately warp someone to an instance after log in? I feel like spawning inside an instance will be super difficult from a technical POV.

**Comment by pandinocoder** [comment]
This should have been filed as a discussion first.
